services/meta/client.go

```diff
@@ -957,35 +957,6 @@ func (c *Client) updateAuthCache() {
 	c.authCache = newCache
 }
 
-// snapshot will save the current meta data to disk
-func snapshot(path string, data *Data) error {
-	file := filepath.Join(path, metaFile)
-	tmpFile := file + "tmp"
-
-	f, err := os.Create(tmpFile)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	var d []byte
-	if b, err := data.MarshalBinary(); err != nil {
-		return err
-	} else {
-		d = b
-	}
-
-	if _, err := f.Write(d); err != nil {
-		return err
-	}
-
-	if err = f.Close(); nil != err {
-		return err
-	}
-
-	return os.Rename(tmpFile, file)
```

Comment:
@jsternberg I tried that, `_unix.go` is not standard GOOS constant. (refer to https://github.com/golang/go/tree/master/src/runtime/internal/sys, look at the zgoos family of files). With `_unix.go` it does compile on Linux, but fails on windows
```go
# github.com/influxdata/influxdb/services/meta
services\meta\sysrename_windows.go:8: sysrename redeclared in this block
        previous declaration at services\meta\sysrename_unix.go:8
```